src/components/Person/Person.jsx

```diff
@@ -1 +1,32 @@
 // export const Person = ({ person }) => ();
+import React from 'react';
+
+export function Person({
+  person: {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  },
+}) {
+  const personPartner = (sex === 'm') ? `wife` : `husband`;
```

Comment:
let's move move each operand to the separate line
see example in second rule in checklist: https://github.com/mate-academy/react_person/blob/master/checklist.md